cmd/node/factory/interface.go

```diff
@@ -19,4 +20,8 @@ type HeaderSigVerifierHandler interface {
 	VerifyRandSeedAndLeaderSignature(header data.HeaderHandler) error
 	VerifySignature(header data.HeaderHandler) error
 	IsInterfaceNil() bool
-}
\ No newline at end of file
+}
+
+type interceptedDataFactoryWithFinalAttesterSetter interface {
```

Comment:
add comment to interface.

Reply:
removed, no need for this at it will be injected